Remotion/Web/Development.WebTesting/Utilities/ProcessUtils.cs

```diff
@@ -143,7 +143,16 @@ public static void GracefulProcessShutdown ([NotNull] Process process, TimeSpan
       }
 
       // Force closing the process
-      process.Kill();
+      try
+      {
+        process.Kill();
+      }
+      catch (Win32Exception)

```

Comment:
One more:
https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.process.kill?view=netframework-4.8

Kill is said to throw an InvalidOpEx if the process has alreayd existed. Please verify.

Reply:
Oh, and we should probably doc all the non-issue exceptions regardless if they are mentioned in the docs but do not apply for non-obvious reasons.

Reply:
That's true, I'll add another `catch` block.